package.json

```diff
@@ -5,13 +5,17 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "postinstall": "patch-package"
   },
   "dependencies": {
     "@emotion/react": "^11.1.5",
```

Comment:
I guess the Emotion dependencies + config can be removed?

Reply:
https://github.com/upleveled/connecting-to-external-apis-secrets-with-stripe-fall-2022-vienna-austria/blob/85f6653d67eeb9ccaf099006071fdc8d8db4124b/package.json#L11-L19